apps/arweave/src/ar_block.erl

```diff
@@ -484,8 +482,7 @@ generate_block_data_segment_base(B) ->
 							integer_to_binary(ScheduledRateDividend),
 							integer_to_binary(ScheduledRateDivisor),
 							integer_to_binary(B#block.packing_2_5_threshold),
-							integer_to_binary(B#block.strict_data_split_threshold)
-							| Props
+							integer_to_binary(B#block.strict_data_split_threshold) | Props
```

Comment:
Is this an elvis change? Personally I prefer to keep newlines consistent within an operator. For this operator we've made the choice to put each list element on its own line, so I would prefer to also put the `| Props` on its own line to be consistent. I've been bitten before by inconsistent new lines where only one element is put on the same line as the others and it gets missed when ssomeone is scannign the code because they expect all elements to be on new lines.

Reply:
the default rule says that there can't be space on the left side of an operator, this has the effect that we have

```erlang
ValA +
ValB +
ValC.
```

and not

```erlang
ValA
+ ValB
+ ValC
```

personally I'm not trained to read code like thie

```js
{
  a: 1
  ,b: 2
  ,c: 3
 }
 ```

Reply:
I'm fine with 
```
ValA +
ValB +
ValC.
```

or 

```
ValA
+ ValB
+ ValC
```

and think in general commas should be treated differently from math operators for the reason you mention (commas generally bind to the left value, but it's less clear with binary math operators which value they more logically bind to - think of grammar school long addition, in the US we're taught to put the `+` at the beginning of every line)

But that said, I retract my original point on the `| Elements` and defer to the elvis rule